src/Resources/config/config.yaml

```diff
@@ -0,0 +1,15 @@
+sylius_grid:
+    templates:
+        action:
+            edit_order: "@SetonoSyliusOrderEditPlugin/admin/order/grid/editOrder.html.twig"
+    grids:
+        sylius_admin_order:
+            actions:
+                item:
+                    delete:
+                        type: edit_order
+                        options:
+                            link:
+                                route: sylius_admin_order_update
+                                parameters:
+                                    id: resource.id
```

Comment:
I would rather add this in an event subscriber

Reply:
I had a question - why you did it with an event subscriber 😄 This way is (of course, it's my opinion) 1000x more readable and straightforward. Also, people in Sylius' ecosystem are used to the yaml configurations. It's also easier to **not** include this change is one decides, as it's simply not importing the configuration file. 

Let's start the discussion :)

Reply:
> It's also easier to not include this change is one decides, as it's simply not importing the configuration file

This is only right if this is the only thing you configure in the file. If it was done with an event subscriber, you could allow to disable the event subscriber in the configuration of the plugin _or_ users could remove it in a compiler pass. But, practically, 99.99% would never touch it.

My reasons for event subscriber:
- I _don't_ have to remind people to add a config file
- I like to put as much config inside PHP files because my tooling can help me if anything changes.
- I am able to unit test an event subscriber. With the config approach you actually need a functional test to test that it's added

But it's not that I am against it as I am with other things. I am just trying to move away from including the `config.yaml` that all plugins do when they could instead avoid it. So the argument for me is probably more like: Why would you want people to include a file if they don't have to _and_ you'd get better DX for yourself not doing it?

Reply:
I understand your approach. IMO the best compromise is having such configurations in php config files 😄 but I do not thing it's worth to argue more about such a trivial thing, no problem for me to return to the event subscriber 👍